### DISCUSSION TITLE
History of Orchard

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#14945] [General]
During the latest triage meeting, we had a funny moment discovering that @MikeAlhayek, one of the most prolific contributors in the recent releases, didn't exactly know who was @bleroy, which has been program manager in the early days of Orchard.

It makes me think that it could interesting to gather some information about the history of the project, how it started, what were the main milestones, ... and may be make a specific page in the documentation.

Along the years, contributing to Orchard helped me to become a better developer and to embrace the open source mindset.

When I look back and see what the community has been able to achieve, I can't thank myself enough to have been a little part of that great project.

If you want to share your experience about how you heared about Orchard and started using it, don't hesitate to share it in the comments.

**Comment by agriffard** [comment]
I found this on Bertrand's blog: 
https://asp-blogs.azurewebsites.net/bleroy/orchard-team-looking-for-a-new-developer

Bonus point to the one that finds the oldest reference to Orchard still available on the web.

**Reply by hishamco** [reply]
I remember there're some other guys in Orchard who originally moved from the Oxite project, I'm not sure if someone still remembers those days

**Comment by ns8482e** [comment]
This old blog ref that inspired me on orchard cms https://weblogs.asp.net/jongalloway/weblogs-asp-net-now-running-on-orchard-and-azure-web-sites?ssp=1&darkschemeovr=1&setlang=en-US&safesearch=moderate
